packages/data-store/src/types/digitalCredential/IAbstractDigitalCredentialStore.ts

```diff
@@ -0,0 +1,34 @@
+import { CredentialCorrelationType, CredentialStateType, DigitalCredential } from './digitalCredential'
+
+export type GetDigitalCredentialArgs = { id: string } | { hash: string }
+
+export type FindDigitalCredentialArgs = Array<Partial<DigitalCredential>>
+
+export type GetDigitalCredentialsArgs = {
+  filter?: FindDigitalCredentialArgs
+  skip?: number
+  take?: number
```

Comment:
Did you doublecheck this with how the cloud wallet expects pagination to happen?

Reply:
updated this impl to be more inline with supabase client handling of sorting/pagination

Reply:
See remark, as this is a different layer, it should not only behave directly like how supabase wants it. The point really was that our interface should be relative easy. For instance if we work with X and limit use that, instead of (from) X and (to) Y.

If the comment on the sort interface is solved we should be good for both